libpod/oci_conmon_common.go

```diff
@@ -165,6 +165,16 @@ func (r *ConmonOCIRuntime) Path() string {
 	return r.path
 }
 
+// Updates OCI runtime name
+func (r *ConmonOCIRuntime) UpdateName(newName string) {
```

Comment:
This is potentially dangerous for `podman system service`. You're changing every call to this Libpod to use the WASM runtime.

Reply:
Instead, consider making a separate instance of the OCI runtime (`crun-wasm` or similar?) which invokes crun with the WASM extension

Reply:
Noted Creating a new `OCIRuntime` instance is better ( intention with PR was to show where toggle will happen ) , I will amend this in final version.